SpatialGDK/Source/SpatialGDK/Private/Utils/Interest/ResultTypes.cpp

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Utils/Interest/ResultTypes.h"
+
+#include "UObject/UObjectIterator.h"
```

Comment:
Nit: unless you are trying to split out engine headers and our headers, alphabetize if you can.

Reply:
I _think_ this is correct- the header for the source, then the split engine and gdk includes. Would rather be consistent, but don't feel too strongly

Reply:
didn't know if there was a policy.